app/jobs/concerns/default_job_behaviour.rb

```diff
@@ -36,4 +37,13 @@ module DefaultJobBehaviour
   def log_failure_to_sentry?(_exception)
     executions <= 4 || executions == MAX_ATTEMPTS
   end
+
+  def job_link
+    @job_link ||= "https://#{domain.host_name}/super_admins/good_job/jobs/#{job_id}"
```

Comment:
Est-ce qu'on peut utiliser un helper rails ici ? ou est-ce que c'est pour limiter le risque d'erreur dans le code qui gère les erreurs qu'on préfère de l'interpolation ?
```suggestion
    @job_link ||= Rails.application.routes.url_helpers.super_admins_good_job_url(job_id, host: domain.host_name)
```

Reply:
je pense que c’est pour faire la bascule `rdv-aide-numerique` => `rdv-solidarites` ?